src/Integration.Vsix.UnitTests/CFamily/VcxProject/FileConfigTests.cs

```diff
@@ -34,6 +35,25 @@ namespace SonarLint.VisualStudio.Integration.UnitTests.CFamily.VcxProject
     public class FileConfigTests
     {
         private readonly TestLogger testLogger = new TestLogger();
+        private const string ClFilePath = "C:\\path\\cl.exe";
+        private const string ClangClFilePath = "C:\\path\\clang-cl.exe";
+
+        private static Mock<IFileSystem> CreateFileSystemWithNoFiles()
```

Comment:
Minor: we started to use Substitute.For<> instead of Mock<> in our tests, if you can, please adopt the change at least for IFileSystem mock